### ISSUE TITLE
[@tailwindcss/language-server] Latest version of the language server is still bundled with tailwind 3.3.0

### ISSUE THREAD

**Issue by weizheheng** [tailwindlabs/tailwindcss-intellisense#892]
Tailwind CSS: v3.4.0
package manager: npm
OS: macOS
@tailwindcss/language-server: 0.0.14

**Describe your issue**

The latest @tailwindcss/language-server 0.0.14 is still bundled with tailwindcss 3.3.0 causing the autocomplete not working for all the new features like the new `size` utility. I am using Neovim, therefore I am only installing the language server but not the vs code extension itself.

Is there any plan to push a new release soon? For the time being I am installing the latest insider build `@tailwindcss/language-server@0.0.0-insiders.3a46554` which is using tailwindcss 3.4.0 and autocomplete works correctly.

Finally, congrats on the release of tailwindcss 3.4.0!

**Comment by jereswinnen** [comment]
I’m having the same issue. Can you tell me how I can install the latest insider build as well? Thanks!

**Comment by weizheheng** [comment]
> I’m having the same issue. Can you tell me how I can install the latest insider build as well? Thanks!

Hi, you can visit https://www.npmjs.com/package/@tailwindcss/language-server/v/0.0.0-insiders.3a46554

```bash
# -g is to install it globally, depending on your need, you can decide to install it globally or not

npm install -g @tailwindcss/language-server@0.0.0-insiders.3a46554
```

**Comment by jereswinnen** [comment]
Thanks!

**Comment by thecrypticace** [comment]
I'll tag a release of the language server that includes v3.4 tomorrow

**Comment by thecrypticace** [comment]
Published as `@tailwindcss/language-server@0.0.15`
